client/src/App.js

```diff
@@ -1,28 +1,45 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
 import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
 import Movie from './Movies/Movie';
+import { Route } from "react-router-dom";
 
 export default class App extends Component {
   constructor() {
     super();
     this.state = {
-      savedList: []
+      savedList: [],
+      movies: []
     };
   }
 
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/api/movies')
+      .then(response => {
+        this.setState(() => ({ movies: response.data }));
+      })
+      .catch(error => {
+        console.error('Server Error', error);
+      });
+      
+  }
+
+
+
   addToSavedList = movie => {
     const savedList = this.state.savedList;
     savedList.push(movie);
     this.setState({ savedList });
   };
 
-  render() {
+  render() {console.log(this.state.movies)
```

Comment:
This is a bit of an interesting placement for a `console.log`... it's certainly not preferred.